docs-elastic/index.asciidoc

```diff
@@ -6,4 +6,6 @@ The {n} allows you to provision Elastic Cloud deployments on any Elastic Cloud p
 
 The provider lets you manage Elastic Cloud deployments as code, and introduce DevOps-driven methodologies to manage and deploy the Elastic Stack and solutions.
 
+NOTE: The provider is not subject to the support SLA. Users can use the Elastic Cloud APIs or user console directly to perform changes as a workaround.
```

Comment:
```suggestion
IMPORTANT: The provider is not subject to the support service level agreement (SLA.) You can use the Elastic Cloud APIs or user console directly to perform changes as a workaround.
```

Can you add some links here, e.g. to the user console or API docs? I didn't suggest them as I don't actually know what we support for the Terraform provider. (If you'd rather I add the links, just comment here with what they are.)